src/main/java/de/focusshift/zeiterfassung/publicholiday/PublicHolidayConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package de.focusshift.zeiterfassung.publicholiday;
+
+import de.focus_shift.jollyday.core.HolidayManager;
+import de.focus_shift.jollyday.core.ManagerParameters;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.net.URL;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Configuration
+class PublicHolidayConfiguration {
+
+    private static final List<String> COUNTRIES = List.of("de", "at", "ch", "gb", "gr", "mt", "it", "hr", "es", "nl");
+
+    @Bean
+    Map<String, HolidayManager> holidayManagerMap() {
```

Comment:
Wir können hier nun die Feiertagsregelungen direkt aus jollyday verwenden und können somit die HolidayManagers einfacher erzeugen und alle 'Holidays_xx.xml' löschen. Diese gibt es nur in der UV, da es dort den 24.12 und 31.12 gibt, welche besonders behandelt werden. Dies ändert sich mit den Betriebsferien und somit können wir hier direkt die aus Jollyday nehmen.